dds/DCPS/DataReaderImpl_T.h

```diff
@@ -11,7 +11,9 @@
 #include "dcps_export.h"
 #include "GuidConverter.h"
 
+#ifndef ACE_HAS_CPP11
```

Comment:
Is there an ACE header that should be included to make sure this is defined?  It is probably already there but better to be explicit about it.

Reply:
There doesn't seem to be an ACE header, but I added an explicit include of <memory> just like it is done in unique_ptr.h.